index.js

```diff
@@ -115,6 +118,32 @@ cm.Item({
   }
 });
 
+cm.Item({
+  label: 'Send to mini player',
+  context: cm.SelectorContext('[href*="vine.co/v/"]'),
+  contentScript: 'self.on("click", function (node, data) {' +
+                 '  self.postMessage(node.href);' +
+                 '});',
+  onMessage: function(url) {
+    const id = getVineId(url);
+    updatePanel({domain: 'vine.co', id: id, src: ''});
+    getVineUrl(url, function(err, streamUrl) {
+      if (!err) updatePanel({src: streamUrl});
+    });
+  }
+});
+
+function getVineId(str) {
```

Comment:
There isn't an issue with the get-video-id package itself, that is the intended behavior from npm.

If a version is declared in package.json, npm will listen to your configuration and install that version. If you want to install a different version you must specify that either in the package.json or in your install command, hence `npm install get-video-id@1.0.0`